lib/mongoid/contextual/geo_near.rb

```diff
@@ -246,7 +246,7 @@ def documents
       #
       # @since 3.0.0
       def results
```

Comment:
should be called `result` instead of `results` just because is returning one element, isn't it?